WIP fleet management documentation (#705)
neo4j/docs-aura: modules/ROOT/pages/fleet-management/overview.adoc

[opts="header",cols="3,1,1,1,1"]
|===
| Capability
| Project Viewer
| Metrics Reader
| Member
| Admin

| View deployments
| {check-mark}
|
| {check-mark}
| {check-mark}

| Create deployments
|
|
|
| {check-mark}

| Rename deployments
|
|
|
| {check-mark}

| Delete deployments
|
|
|
| {check-mark}

| View connection URL
| {check-mark}
|
| {check-mark}
| {check-mark}

| Add connection URL
|
|
|
| {check-mark}

| Edit connection URL
|
|
|
| {check-mark}

| View token info (existence, active)
| {check-mark}
|
| {check-mark}
| {check-mark}

| Create token
|
|
|
| {check-mark}

| Regenerate token
|
|
|
| {check-mark}

| Delete token
|
|
|
| {check-mark}
|===

== Features

Fleet management offers various features to manage and monitor your self-managed Neo4j deployments:

* Metrics
* Topology views (graph and tabular format)
* Server configuration viewer
* Database metadata
* License status viewer

In addition to these features, the user tools Query, Explore, and Dashboards (available in the Aura console) are also available for self-managed deployments.
See xref:managing-instances/self-managed.adoc[Self-managed instances] for more information.

[NOTE]
====

> User tools connect to your deployment using their own bolt connection, independent of the fleet management plugin.

Author: I would write it like this:

```suggestion
Connecting to self-managed instances with user tools works independently of fleet management plugin and is therefore available for unmonitored deployments as well.
```

= Overview

Get an enterprise-wide view of all your Neo4j instances, both Aura instances and/or self-managed.

For users operating in hybrid environments, it’s common to have a mix of Aura instances and self-managed Neo4j deployments. 
Integrating self-managed deployment into the Aura console allows for a more centralized management.

== Centralized management
Registering self-managed Neo4j deployments in the Aura Console allows you to monitor all your Neo4j databases in one place.

== Security 

Fleet management is designed to be secure, no inbound connections to your self-managed deployment are allowed.
Further, there is no capability for performing any write operations or other mutations remotely.

No data is read from your databases.
See xref:fleet-management/data.adoc[Data transparency] for a complete list of the data sent to Aura from your self-managed deployment.

== Features

Fleet management offers various features to manage and monitor your self-managed Neo4j deployments:

* Metrics
* Topology views (graph and tabular format)
* Server configuration viewer
* Database metadata
* License status viewer

> In addition to these features, the user tools Query, Explore, and Dashboards (available in the Aura console) are also available for self-managed deployments.

Reviewer: @AlexicaWright Your previous comment about write actions disappeared, but the user tools use their own bolt connections. All we provide to them is the URL for connecting. So they can perform writes as well, but the customer must configure their firewall accordingly, and provide credentials when connecting.

Reviewer 2: I see, so it's only from an EFM perspective that no writes are allowed?

Reviewer: Exactly. The EFM connection cannot write anything. Using the user tools involves establishing a separate bolt connection, which does not come with that limitation.

Author: Jessica will add a link to the new page describing connecting to self managed instances.

= Overview

> Get an enterprise-wide view of all your Neo4j systems—Aura-managed or self-managed.

Reviewer 2: ```suggestion
Get an enterprise-wide view of all your Neo4j instances, both Aura instances and/or self-managed.
```

> For enterprise customers operating in hybrid environments, it’s common to have a mix of *Aura-managed* and *self-managed* Neo4j deployments. By integrating self-managed deployments into the Aura Console, organizations can simplify operations, reduce overhead, and plan for full migration when ready.

Reviewer 2: ```suggestion
For users operating in hybrid environments, it’s common to have a mix of Aura instances and self-managed Neo4j deployments. 
Integrating self-managed deployment into the Aura console allows for a more centralized management.
```

Reviewer: Maybe it's redundant to say it's common to have a mix in hybrid environments, because that is the very definition of hybrid..

> == *Centralized management*

Reviewer 2: ```suggestion
== Centralized management
```

> Register self-managed Neo4j deployments directly in the Aura Console to:

Reviewer 2: ```suggestion
Registering self-managed Neo4j deployments in the Aura Console allows you to monitor all your Neo4j databases in one place.
```

* Monitor and manage all Neo4j databases across your organization in one place
* Monitor health and performance metrics

> * Run Cypher queries, explore graphs, and view dashboards

Reviewer 2: ```suggestion

```

> == *Security first*

Reviewer 2: ```suggestion
== Security 
```

Fleet management is designed with security first.
Your deployment will reach out to the Aura console to send carefully considered, non-sensitive data, but we purposely designed the system never to allow inbound connections to your self-managed deployment.

And there is *no* capability for performing any write operations or other mutations remotely.

> We never read any data from your databases. A complete listing of the data sent to Aura is documented on the xref:fleet-management/data.adoc[Data transparency] page.

Reviewer 2: ```suggestion
Fleet management is designed to be secure, no inbound connections to your self-managed deployment are allowed.
Further, there is no capability for performing any write operations or other mutations remotely.

No data is read from your databases.
See xref:fleet-management/data.adoc[Data transparency] for a complete list of the data sent to Aura from your self-managed deployment.
```

== Features

> Fleet management offers a wealth of features to manage and monitor your self-managed Neo4j deployments:

Reviewer 2: ```suggestion
Fleet management offers various features to manage and monitor your self-managed Neo4j deployments:
```

* Metrics
* Topology views (graph and tabular format)
* Server configuration viewer
* Database metadata
* License status viewer

> In addition to these features, the following user tools are available for self-managed deployments:

Reviewer 2: The use of Query must come with the caveat that no write operations are allowed then?

* Query
* Explore

> * Dashboards

Reviewer 2: ```suggestion
In addition to these features, the user tools Query, Explore, and Dashboards (available in the Aura console) are also available for self-managed deployments.
```

= Overview
:description: This page provides an overview of fleet management.

Get an enterprise-wide view of all your Neo4j instances, both Aura instances and self-managed.

> With fleet management, you can monitor your self-managed Neo4j deployments directly from the Aura console.

Reviewer 2: ```suggestion
Fleet management allows you to monitor *all* of your Neo4j instances, both Aura and self-managed, directly from the Aura console.
```

== Architecture

> 

Reviewer 2: ```suggestion
```

Reviewer 2: It feels superfluous to have this as its own section. The content fits nicely in the introduction.

Fleet management uses a plugin that you install on your self-managed Neo4j deployments.
The plugin collects information from your deployment and securely transmits this data to Aura.

See xref:fleet-management/setup.adoc[Add a deployment] for instructions on how to set up the plugin.

== Compatibility

Fleet management works with all self-managed Neo4j deployments running Neo4j 4.4 and later.
All operating systems and deployment modalities supported by Neo4j are also supported by fleet management.

> Community Edition deployments are subject to some limitations due to the limited set of features, such as no metrics being available.

Reviewer 2: If no metrics are available, then what is?

[NOTE]
====
Community Edition compatibility is offered on a best-effort basis. Neo4j does not offer any technical support for it.
====

== Security 

Fleet management is designed to be secure, no inbound connections to your self-managed deployment are allowed.
Further, there is no capability for performing any write operations or other mutations remotely.

No data is read from your databases.
See xref:fleet-management/data.adoc[Data transparency] for a complete list of the data sent to Aura from your self-managed deployment.

== Features

Fleet management offers various features to manage and monitor your self-managed Neo4j deployments:

* Metrics
* Topology views (graph and tabular format)
* Server configuration viewer
* Database metadata
* License status viewer

> In addition to these features, the user tools Query, Explore, and Dashboards (available in the Aura console) are also available for self-managed deployments.

Reviewer 2: I wonder if we should even mention this here? It is unrelated to fleet management and will be covered under _Manage instances_ or something similar?
